packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -46,6 +47,8 @@ export interface AlertProps extends Omit<React.HTMLProps<HTMLDivElement>, 'actio
   truncateTitle?: number;
   /** Position of the tooltip which is displayed if text is truncated */
   tooltipPosition?: 'auto' | 'top' | 'bottom' | 'left' | 'right';
+  /** Set a custom icon to the Alert */
```

Comment:
Can you also expand on the description here `If not set the icon is set according to the variant`